recipes-wpe/wpeframework/wpeframework-tools-native_git.bb

```diff
@@ -0,0 +1,28 @@
+SUMMARY = "Host/Native tooling for the Web Platform for Embedded Framework"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=1fe8768cbb5fd322f7d50656133549de"
+FILESEXTRAPATHS_prepend := "${THISDIR}/wpeframework:"
+
+PR = "r0"
+PV = "3.0+git${SRCPV}"
+S = "${WORKDIR}/git"
+
+SRC_URI = "git://github.com/WebPlatformForEmbedded/WPEFramework.git;protocol=git;branch=master \
+           file://0001-Changes-to-support-JSON-and-ProxyStub-generator-with-python3.patch \
+"
+
+SRC_URI[md5sum] = "42b518b9ccd6852d1d709749bc96cb70"
+SRC_URI[sha256sum] = "f3c45b121cf6257eafabdc3a8008763aed1cd7da06dbabc59a9e4d2a5e4e6697"
+
+SRCREV = "4cf3f44509c7d1ad94c897766a0efa9846efbe6f"
+
+inherit cmake pkgconfig native python3native
```

Comment:
AFAIK this package has nothing to do with python during the bake. Python is needed to run this package so it's needed on wpeframework package level only

Reply:
if that is so, then add RDEPENDS_${PN} = "python3"